Automattic/newspack-newsletters
# fix: update translation text domains

### All Submissions:


### Changes proposed in this Pull Request:

This PR fixes a handful of textdomain errors in the plugin, where strings were incorrectly using `newspack` instead of `newspack-newsletters`. 

Since this plugin is hosted on WP.org, no translation file changes are needed in it, as they're handled on translate.wordpress.org.

See: 1200550061930446-as-1205587112319820

### How to test the changes in this Pull Request:

1. Spot check the changes, and confirm they're just switching `newspack` to `newspack-newsletters` where it appears in strings marked for translation.

### Other information:
